ios/Classes/ReactiveBle/Tasks/PeripheralTaskRegistry/PeripheralTaskController.swift

```diff
@@ -2,14 +2,14 @@ protocol PeripheralTaskController {
 
     associatedtype TaskSpec: PeripheralTaskSpec
 
-    typealias Task = PeripheralTask<TaskSpec>
+    typealias PTask = PeripheralTask<TaskSpec>
```

Comment:
Single-letter prefix is too obscure. Consider using `TaskType` (not ideal, but), or `SubjectTask`, or something else that reads fine and makes sense. Please, use the chosen approach consistently throughout the rest of the code.

Reply:
I renamed `PTask` to `SubjectTask` as you suggested.